src/main/java/nl/cwts/networkanalysis/Clustering.java

```diff
@@ -152,6 +153,43 @@ public int getNClusters()
     }
 
     /**
+     * Returns whether cluster is empty or not.
+     *
+     * @return Cluster is empty for each cluster
+     */
+    public boolean[] getClusterIsNotEmpty()
+    {
+        boolean[] clusterNotEmpty = new boolean[nClusters];
```

Comment:
The style in the rest of the package is to define and initialize variables at different lines. So, perhaps we should do this also in this new method.